### ISSUE TITLE
postRideCallback runs whether cancelled or complete - is there a way of distinguishing between these?

### ISSUE THREAD

**Issue by awidgery** [zurb/joyride#203]

**Comment by robinfhu** [comment]
Yes.  The `postRideCallback` is passed three arguments. The last argument is `isAborted`.  The `isAborted` flag is set to true if the tutorial was cancelled via the close button.

**Comment by raffaelmeier** [comment]
can you specify how to use that function (isAborted) in a short example? I am not managing to get it to work... thanks in advance...

**Comment by raffaelmeier** [comment]
$(window).load(function() {
              $("#joyRideTipContent").joyride({
                  autoStart : true,
				  postRideCallback :function() { if (this.isAborted == true ) { window.alert('hello world');  } // or whatever you want to call
        						}
              });
            });
 does not work
